modules/acknowledgements/js/acknowledgements_helper.js

```diff
@@ -1,25 +1,195 @@
 function hideFilter(obj) {
```

Comment:
Since you are rewriting the module, you should get rid of all the `helper` functions that were there before, and redo it in `ES6` react way